nexus/src/app/vpc_subnet.rs

```diff
@@ -184,69 +215,25 @@ impl super::Nexus {
         }
     }
 
-    pub async fn vpc_list_subnets(
+    pub async fn vpc_subnet_list(
```

Comment:
More a question than anything. I noticed changing the ordering of this name to reflect the public API method names, but the `vpc_update_subnet` function below (and others) weren't changed the same way. What's the reasoning behind the naming at this layer?

Reply:
I've got a follow up to fully correct all of these I just didn't do it all in this one PR to reduce the noise. Overall my intention here is to have a consistent naming scheme. I'll open up a broader PR for the renames and we can discuss in detail if it's the right move there.

Reply:
Sounds good, and like you're on top of it. Thanks!